cmd/hstspreload/main.go

```diff
@@ -116,6 +116,18 @@ includeSubDomains: %s%t%s
 				underline, domain, resetFormat,
 				bold, state.Mode, resetFormat,
 				bold, state.IncludeSubDomains, resetFormat)
+		} else if status == preloadlist.AncestorEntryFound {
+			fmt.Printf(`%s%s%s is preloaded by virtue of its ancestor domain:
+
+         ancestor: %s%s%s
```

Comment:
I think this is a good idea, and I think we should actually show the found domain all the time, like chrome://net-internals/#hsts

Could you
1) change `ancestor` to `domain` to match net-internals?
2) factor out the entry printing logic, or share it with with the normal one?

Reply:
Seems to me that it is cleaner to just combine ExactEntryFound and AncestorEntryFound output log together. The API already exposes the enum for users to distinguish these two cases, and the command line tool also shows you the actual domain entry found in the preload list. net-internals does this as well. There is no need to specifically say if a domain is found by its ancestor because that information is implicit.